topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -56,9 +56,8 @@ async def add_metadata(self, item: Item) -> None:
             "linz:when_scanned": item_dict["when_scanned"],
             "linz:photo_version": item_dict["photo_version"],
         }
-        item.stac_item.properties.update(properties)
-        item.stac_item.id = item_dict["sufi"]
-        item.asset_basename = f"{item_dict['survey']}/{item_dict['sufi']}"
+        item.properties.update(properties)
+        item.id_ = item_dict["sufi"]
```

Comment:
Will this throw if it cannot be found here?

Reply:
We added item.is_valid logic here to catch errors